lib/RPM/Grill.pm

```diff
@@ -101,6 +104,31 @@ $Is_64bit{$_} = 0  for @Non_RPM_Arches;
 # BEGIN data-gathering code
 
 #
+# Loads blacklisted test cases from file passed as the first and only argument
+# into the %blacklisted_tests global associative array.
+sub load_blacklist {
+    my $blacklist = shift;
+
+    if ($blacklist eq '') {
+        return;
+    }
+
+    open my $fh, '<', $blacklist
+        or die "Unable to open $blacklist: $!";
+    my @tests = <$fh>;
+
+    # Create associative array with blacklisted tests
+    for my $el (@tests) {
```

Comment:
1. IMHO it is better to use a format like yaml to store the whitelisted tests
2. Perl gives you better way to grep and map array to hash e.g: `$hash{ chomp $_ }++ for (@array);`